Misc/NEWS.d/next/Library/2025-05-18-07-25-15.gh-issue-134173.53oOoF.rst

```diff
@@ -0,0 +1,4 @@
+Add ``_get_snapshot()`` method to :class:`concurrent.futures.Future` to
+atomically retrieve all future state in a single lock acquisition. This speeds
+up :mod:`asyncio`'s ``_copy_future_state()`` by up to 4x when transferring state

```

Comment:
No need to mention the internal details, just mention that it helps in performance of executing in thread pool

Reply:
adjusted